scripts/start.sh

```diff
@@ -19,7 +19,6 @@ MODEL_DIR="$(cd "test/resources/models" > /dev/null && pwd)"
 $docker_command run \
     -v "$MODEL_DIR":/opt/ml/model:ro \
     -p 8080:8080 \
-    -e "SAGEMAKER_TFS_DEFAULT_MODEL_NAME=half_plus_three" \
```

Comment:
not clear what this change has to do with ping

Reply:
Added it back. I removed it for local testing and forgot to add it back.